core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/audits/responses/AuditEventV4Responses.java

```diff
@@ -0,0 +1,29 @@
+package com.sequenceiq.cloudbreak.api.endpoint.v4.audits.responses;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import io.swagger.annotations.ApiModel;
+
+@ApiModel
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class AuditEventV4Responses {
```

Comment:
please remove this, i already created general set response for this in the another pr